packages/transport-bull/src/cleaner.js

```diff
@@ -1,30 +1,28 @@
 
-import RedisExclusiveTask from 'redis-exclusive-task';
+import Queue from 'bull';
 
 import { DEFAULT_INTERVAL } from './intervals';
 
-let subscriberCleanerRunning = false;
-
 function runSubscriberCleaning(transport, log) {
-  if (subscriberCleanerRunning) {
-    return;
-  }
-
-  subscriberCleanerRunning = true; // eslint-disable-line better-mutation/no-mutation
-  log.info(`runSubscriberCleaning`);
-  RedisExclusiveTask.configure([transport.redis.newConnection()], log);
-
-  RedisExclusiveTask.run(
-    'stlr:subscribers:cleaner',
-    () => transport._cleanResources(),
-    DEFAULT_INTERVAL
-  );
-
-  RedisExclusiveTask.run(
-    'stlr:queues:remover',
-    () => transport._removeUnusedQueues('stlr:*:req'),
-    DEFAULT_INTERVAL
-  );
+  const queue = new Queue('stlr:periodicTasks', transport.redis.newConnection());
+  const cron = `*/${DEFAULT_INTERVAL} * * * *`; // Every X minutes
+  const repeatableJobOptions = {
+    repeat: {
+      cron,
+    },
+    removeOnFail: false,
+    removeOnComplete: true,
+  };
+
+  const CLEAN_SUBSCRIBERS_JOB_NAME = 'stlr:subscribers:cleaner';
+  queue.process(CLEAN_SUBSCRIBERS_JOB_NAME, transport._cleanResources());
+  queue.add(CLEAN_SUBSCRIBERS_JOB_NAME, { action: 'run' }, repeatableJobOptions);
+  log.info(`@${CLEAN_SUBSCRIBERS_JOB_NAME}: job is scheduled to run with CRON expression ${cron}`);
+
+  const CLEAN_USUSED_QUEUES_JOB_NAME = 'stlr:queues:remover';
+  queue.process(CLEAN_USUSED_QUEUES_JOB_NAME, transport._removeUnusedQueues('stlr:*:req'));
+  queue.add(CLEAN_USUSED_QUEUES_JOB_NAME, { action: 'run' }, repeatableJobOptions);
+  log.info(`@${CLEAN_USUSED_QUEUES_JOB_NAME}: job is scheduled to run with CRON expression ${cron}`);
 }
```

Comment:
could so something nicer and setup outside of the function a object
```
const jobs = {
    'stlr:subscribers:cleaner': () => transport._cleanResources(),
    'stlr:queues:remover': () => transport._removeUnusedQueues('stlr:*:req')
}
```

and then forEach over it